content/blog/2022/eddy-symposium-report/index.md

```diff
@@ -0,0 +1,44 @@
+
+## 2i2c's role
+
+2i2c, with input from Symposium CoChair's [Dan Marsh](https://staff.ucar.edu/users/marsh) and [Ryan McGranaghan](http://www.ryanmcgranaghan.com/about.html), rapidly deployed a cloud-hosted JupyterHub for use during the event. The work to set up that hub, openly chronicled in [this GitHub issue](https://github.com/2i2c-org/infrastructure/issues/1329), included swapping out a Pangeo-style software environment for a heliophysics-specific resource developed by [HelioCloud](http://heliocloud.org/) with special thanks to [Brian Thomas](https://www.nasa.gov/offices/oce/bios/mwm/Thomas-Brian-bio.html)!
+
+2i2c CoFounder Fernando Pérez gave a talk on how he is ["living la vida nube"](https://docs.google.com/presentation/d/1i99eQqOVCSQyxoyscUo0dl3Ybgdex5qKo_KEO9YGxQg/edit?usp=drivesdk) . Fernando described the ways he, research collaborators, and students are using the Jupyter ecosystem. Diverse and curated tools in Jupyter hubs for the [Jupyter Meets the Earth Project](https://jupytearth.org/) and [Berkeley's data science programs](https://data.berkeley.edu/) were highlighted. The talk showcased how these tools have been integrated to support individuals and communities of practice in data-driven research. In response to requests from the organizers and participants, Fernando gave a demonstration on how to use the hub 2i2c set up for the Symposium and an introduction to version control using `git`.
+
+2i2c CoFounder Jim Colliander gave a talk titled [Governing the Science Commons](https://bit.ly/eddy-science-commons). Three key points from Jim's talk were: the virtue that should guide the improvements to the scientific enterprise is intellectual generosity; implementation of intellectual generosity into science requires commons-based governance; the convergence of open source tools that support data-intensive collaborative research and learning (as showcased by Fernando) and agency interest ([NASA TOPS](https://science.nasa.gov/open-science/transform-to-open-science), [UNESCO](https://unesdoc.unesco.org/ark:/48223/pf0000379949/PDF/379949eng.pdf.multi.page=1)) in open science is an inflection point for global change.
+
+
+## Things we learned
+
+Our experience with the Symposium taught 2i2c a few things. 
+
+We learned that our engineering team can rapidly deploy interactive computing resources to support a research and education community. Along the way, we confirmed what we've been learning from Pangeo and the neuroscience communities: flexible methods to customize the software environment are necessary. We confirmed that our developing shared responsibility model, enabling domain-specific experts to provide curated toolchains for their communities while leveraging 2i2c's infrastructure expertise, is the right approach.
```

Comment:
`shared responsibility model` could cross-link to our 2i2c documentation on this